pkgs/terraform-config-inspect/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "terraform-config-inspect";
```

Comment:
@davido-nw I think in this case we should just stick to the nixpkgs standards: https://nixos.org/manual/nixpkgs/unstable/#sec-package-naming specifically:

> If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23".

This way we dont have to worry about a fork 🙂  But you can still refer to it as `terraform-config-inspect` everywhere else like in `overlay`

Reply:
Is the `version` and `rev` fine (considering I had to basically make them up)?

Reply:
`version` is close but it should be with dashes to the spec. And just make `rev` the commit hash:

```
version = "2021-04-27"
rev = <commit hash>
```

And you can point it back to the upstream source repo